src/Orc.CsvTextEditor/Operations/AddColumnOperation.cs

```diff
@@ -7,6 +7,62 @@
 
 namespace Orc.CsvTextEditor.Operations
 {
+    internal class QuoteColumnOperation : OperationBase
+    {
+        #region Constructors
+        public QuoteColumnOperation(ICsvTextEditorInstance csvTextEditorInstance)
+            : base(csvTextEditorInstance)
+        {
+        }
+        #endregion
+
+
+        public override void Execute()
+        {
+            var location = CsvTextEditorInstance.GetLocation();
+            var startPosition = location.Column.Offset + location.Line.Offset;
+            var endPosition = startPosition + location.Column.Width;
+
+            var text = CsvTextEditorInstance.GetText();
+            text = TryRemoveQuoteFromPosition(endPosition - 2, text, out var quotesRemoved);
```

Comment:
add logging:

```
if (TryRemoveQuoteFromPosition(endPosition - 2, text, out var updatedText))
{
    Log.Debug($"Removed quotes at end of text @ '{location}'");

    text = updatedText;
}
```

Reply:
maybe it would be better to log whole operation